auth.ts

```
@@ -0,0 +1,75 @@
+import type { JWT } from 'next-auth/jwt';
+import GoogleProvider from 'next-auth/providers/google';
+
+import { createAuthClient } from '@/app/_libs/auth-client';
+import type { IAuthToken } from '@/app/_types/auth';
+import { URLS } from '@/app/constants';
+
+export const authOptions: AuthOptions = {
+  secret: process.env.NEXTAUTH_SECRET,
+  providers: [
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)